### ISSUE TITLE
No Docker image for v7 RC

### ISSUE THREAD

**Issue by Casuallynoted** [Chocobozzz/PeerTube#6740]
### Describe the problem

At the moment, it doesn't seem like a Docker image is available for v7 RC, which was released this morning. If it's possible, I'd love to see a tag for that image for folks using docker.

**Comment by Chocobozzz** [comment]
Hi,

We don't provide RC Docker versions but you can use the develop one, that is sync with the RC for the moment

**Comment by LiFaytheGoblin** [comment]
@Chocobozzz are there docker images for past versions? I have 3.1.0 and want to update step by step to the latest version  (to 3.3.0 for now instead of going to the latest image directly)

**Comment by Chocobozzz** [comment]
Yes, you can see the containers on https://hub.docker.com/r/chocobozzz/peertube/tags

**Comment by LiFaytheGoblin** [comment]
> Yes, you can see the containers on https://hub.docker.com/r/chocobozzz/peertube/tags

Oh perfect, thanks, funny I didn't find this before!

**Comment by LiFaytheGoblin** [comment]
> Yes, you can see the containers on https://hub.docker.com/r/chocobozzz/peertube/tags

Ah, I guess I was confused because this is the buster image, but I was looking for the webserver image!
